### PR TITLE
Documentation updates for upcoming 3.9 release

### PR DESCRIPTION
**Pull request by mato**
To be merged on Monday, once mirage 3.9.0 has been merged and propagated to OPAM.

/cc @mirage/core

### CODE REVIEW

**Review by talex5 — APPROVED** [review]

**Review by dinosaure — APPROVED** [review]

File: tmpl/blog/announcing-mirage-39-release.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# MirageOS 3.9.0 release
+
+We are pleased to announce the release of MirageOS 3.9.0.
+
+Our last release announcement was for [MirageOS 3.6.0](https://mirage.io/blog/announcing-mirage-36-release), so we will also cover changes since 3.7.x and 3.8.x in this announcement.
+
+New features:
+
+- The Xen backend has been re-written from scratch to be based on Solo5, and now supports PVHv2 on Xen 4.10 or higher, and QubesOS 4.0.
+- As part of this re-write, the existing Mini-OS based implementation has been retired, and all non-UNIX backends now use a unified OCaml runtime.
```
Comment (avsm) [inline tmpl/blog/announcing-mirage-39-release.md:10]:
```suggestion
- As part of this re-write, the existing Mini-OS based implementation has been retired, and all non-UNIX backends now use a unified OCaml runtime based on `ocaml-freestanding`.
```

File: tmpl/blog/announcing-mirage-39-release.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# MirageOS 3.9.0 release
+
+We are pleased to announce the release of MirageOS 3.9.0.
+
+Our last release announcement was for [MirageOS 3.6.0](https://mirage.io/blog/announcing-mirage-36-release), so we will also cover changes since 3.7.x and 3.8.x in this announcement.
+
+New features:
+
+- The Xen backend has been re-written from scratch to be based on Solo5, and now supports PVHv2 on Xen 4.10 or higher, and QubesOS 4.0.
```
Comment (avsm) [inline tmpl/blog/announcing-mirage-39-release.md:9]:
```suggestion
- The Xen backend has been [re-written from scratch](https://github.com/mirage/mirage/issues/1159) to be based on Solo5, and now supports PVHv2 on Xen 4.10 or higher, and QubesOS 4.0.
```

File: tmpl/wiki/install.md [hunk 3]
```diff
@@ -82,11 +82,11 @@ After installation, `opam update -u` refreshes the package list and recompiles p
     # list of your remotes, which should include opam.ocaml.org
     $ opam remote
 
-Next, make sure you have at least **OCaml 4.07.0 or higher** as your active compiler. This is generally the case on MacOS X, though Debian only has it in the *testing* distribution at present. But don't worry: if your compiler is out of date, just run `opam switch` to have it locally install the right version for you.
+Next, make sure you have at least **OCaml 4.08.0 or higher** as your active compiler. This is generally the case on MacOS X, though Debian only has it in the *testing* distribution at present. But don't worry: if your compiler is out of date, just run `opam switch` to have it locally install the right version for you.
```
Comment (avsm) [inline tmpl/wiki/install.md:85]:
```suggestion
Next, make sure you have at least **OCaml 4.08.0 or higher** as your active compiler. This is generally the case on macOS, though Debian only has it in the *testing* distribution at present. But don't worry: if your compiler is out of date, just run `opam switch` to have it locally install the right version for you.
```

File: tmpl/wiki/libvirt-on-cubieboard.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+#### Warning: OUT OF DATE
```
Comment (avsm) [inline tmpl/wiki/libvirt-on-cubieboard.md:1]:
```suggestion
#### Warning: Out of date
```

No need to shout :-)

**Review by avsm — APPROVED** [review]
Just minor comments

File: tmpl/wiki/xen-on-cubieboard2.md [hunk 5]
```diff
@@ -17,8 +17,7 @@ The remainder of this guide covers:
 
 ## Warning: OUT OF DATE
```
Comment (hannesm) [inline tmpl/wiki/xen-on-cubieboard2.md:18]:
```suggestion
## Warning: Out of date
```

**Review by hannesm — APPROVED** [review]

**Pull request merged**